dev/bots/run_command.dart

```diff
@@ -113,7 +113,6 @@ Future<Command> startCommand(String executable, List<String> arguments, {
         switch (outputMode) {
           case OutputMode.print:
             print(line);
-            break;
           case OutputMode.capture:
             break;
```

Comment:
Why don't remove ones like this?

Reply:
If the case is empty, you still need the break. Otherwise, it would fall-through to the next case.